specification/_json_spec/inference.chat_completion_inference.json

```
@@ -0,0 +1,31 @@
+{
+  "inference.chat_completion_unified": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/chat-completion-inference.html",
+      "description": "Perform chat completion inference"
+    },
+    "stability": "stable",
+    "visibility": "public",
+    "headers": {
+      "accept": ["text/event-stream"],
+      "content_type": ["application/json"]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_inference/chat_completion/{inference_id}/_unified",
```

Shouldn't this be _stream

```suggestion
          "path": "/_inference/chat_completion/{inference_id}/_stream",
```

---

No, it does have `_unified` at the end. I think technically we could remove it since the client code doesn't need the results to be in a specific format for SSE.